lib/modules/commentPreview.js

```diff
@@ -1,7 +1,7 @@
 addModule('commentPreview', {
 	moduleID: 'commentPreview',
 	moduleName: 'Live Comment Preview',
```

Comment:
This module should really be renamed "Live Snudown Preview" since it applies to posts, wiki, and messages in addition to comments.